### PR TITLE
Add docs for new Touchline SL integration

### PR DESCRIPTION
**Pull request by jnsgruk**
## Proposed change

Adds documentation for a new Roth Touchline SL integration proposed in Home Assistant Core https://github.com/home-assistant/core/pull/124557

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/124557
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/5797
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Introduced comprehensive documentation for integrating the Roth Touchline SL underfloor heating system with Home Assistant.
	- Enabled control over Roth Touchline SL climate platform, allowing monitoring and adjustment of heating zones.
	- Enhanced automation capabilities through the assignment of climate entities to a "Global Schedule."

### CODE REVIEW

File: source/_integrations/touchline_sl.markdown [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+title: Roth Touchline SL
+description: Instructions on how to integrate Roth Touchline SL within Home Assistant.
+ha_category:
+  - Climate
+ha_release: 2024.09
```
Comment (joostlek) [inline source/_integrations/touchline_sl.markdown]:
```suggestion
ha_release: 2024.9
```

File: source/_integrations/touchline_sl.markdown [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+ha_iot_class: Cloud Polling
+ha_domain: touchline_sl
+ha_platforms:
+  - climate
+ha_integration_type: integration
+---
+
+The `touchline_sl` climate platform enables you to control [Roth Touchline SL](https://www.roth-uk.com/products/control-systems/roth-touchliner-sl-wireless-system) underfloor heating systems.
```
Comment (joostlek) [inline source/_integrations/touchline_sl.markdown]: It's an integration, not a platform

File: source/_integrations/touchline_sl.markdown [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+## Prerequisites
+
+You must have an account registered with the [Roth Touchline SL dashboard](https://roth-touchlinesl.com/login).
+
```
Comment (c0ffeeca7) [inline source/_integrations/touchline_sl.markdown:19]:
```suggestion

{% include integrations/config_flow.md %}

```
Let's move this directly after the prereqs.

File: source/_integrations/touchline_sl.markdown [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+## Entities
+
+The integration will present each Roth Touchline SL zone as a climate entity, which can:
+
+- Display the current temperature
+- Display the current humidity
+- Set a target temperature
+- Assign to a configured "Global Schedule" using Home Assistant climate entity presets.
+
+{% include integrations/config_flow.md %}
```
Comment (c0ffeeca7) [inline source/_integrations/touchline_sl.markdown]:
```suggestion
```

File: source/_integrations/touchline_sl.markdown [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+  - Climate
+ha_release: 2024.9
+ha_iot_class: Cloud Polling
+ha_domain: touchline_sl
+ha_platforms:
+  - climate
+ha_integration_type: integration
+---
+
+The `touchline_sl` climate platform enables you to control [Roth Touchline SL](https://www.roth-uk.com/products/control-systems/roth-touchliner-sl-wireless-system) underfloor heating systems.
```
Comment (c0ffeeca7) [inline source/_integrations/touchline_sl.markdown]:
```suggestion
The **Roth Touchline SL** climate integration enables you to control [Roth Touchline SL](https://www.roth-uk.com/products/control-systems/roth-touchliner-sl-wireless-system) underfloor heating systems.
```

**Review by c0ffeeca7 — DISMISSED** [review]
Thank you, @jnsgruk :+1:

**Comment by c0ffeeca7** [comment]
awaits changes in related brands PR

File: source/_integrations/touchline_sl.markdown (hunk 2 above)
Comment (jnsgruk) [reply source/_integrations/touchline_sl.markdown]: Fixed!

File: source/_integrations/touchline_sl.markdown (hunk 1 above)
Comment (jnsgruk) [reply source/_integrations/touchline_sl.markdown]: Fixed!

File: source/_integrations/touchline_sl.markdown [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+  - Climate
+ha_release: 2024.9
+ha_iot_class: Cloud Polling
+ha_domain: touchline_sl
+ha_platforms:
+  - climate
+ha_integration_type: integration
+---
+
+The `touchline_sl` climate integration enables you to control [Roth Touchline SL](https://www.roth-uk.com/products/control-systems/roth-touchliner-sl-wireless-system) underfloor heating systems.
```
Comment (c0ffeeca7) [inline source/_integrations/touchline_sl.markdown]:
```suggestion
The **Roth Touchline SL** climate integration enables you to control [Roth Touchline SL](https://www.roth-uk.com/products/control-systems/roth-touchliner-sl-wireless-system) underfloor heating systems.
```
we use the full name as it is shown in the UI.

**Review by c0ffeeca7 — APPROVED** [review]

**Pull request merged**